server/interfaces/hmppsUser.ts

```diff
@@ -0,0 +1,60 @@
+export type AuthSource = 'nomis' | 'delius' | 'external' | 'azuread'
+
+/**
+ * These are the details that all user types share.
+ */
+export interface BaseUser {
+  authSource: AuthSource
+  username: string
+  userId: string
+  name: string
+  displayName: string
+  userRoles: string[]
+  token: string
+}
+
+/**
+ * Prison users are those that have a user account in NOMIS.
+ * HMPPS Auth automatically grants these users a `ROLE_PRISON` role.
+ * Prison users have an additional numerical staffId. The userId is
+ * a stringified version of the staffId. Some teams may need to separately
+ * retrieve the user case load (which prisons that a prison user has access
+ * to) and store it here, an example can be found in `hmpps-prisoner-profile`.
+ */
+export interface PrisonUser extends BaseUser {
+  authSource: 'nomis'
+  staffId: number
+  activeCaseLoadId: string
```

Comment:
This file copied from the template app, just with the addition of `activeCaseLoadId` for `PrisonUser`s